src/pages/index.js

```diff
@@ -19,32 +18,47 @@ import PageNotFound from "../components/pages/PageNotFound";
 
 class IndexPage extends Component {
   state = {
-    userId: generateUUID(),
-    scenario: null,
-    profiles: [],
-    backgroundQuestionIds: [],
-    errors: {},
-    page: "home",
-    workerId: "",
-    email: "",
-    acceptedTerms: false,
-    background: {
+    user: {
+      userId: generateUUID(),
+      workerId: "",
+      email: "",
+      acceptedTerms: false,
       age: "",
       race: "",
       ethnicity: "",
       gender: "",
       education: "",
-      income: "",
+      momeducation: "",
```

Comment:
All variables should be camelCased. This is a JS standard and helps avoid errors.